src/platform/lumin-runtime/platform-factory.js

```diff
@@ -28,6 +30,19 @@ export class PlatformFactory extends NativeFactory {
     this.elementBuilders = {};
     this.controllerBuilders = {};
 
+    // { prismId: { nodeId: { eventName: callbackId } }
+    //
+    // example:
+    // PrismId = 1, ChildNodeId = 1, EventName = `onActivate`
+    // callbackIdObject = onActivateSub(callback)
+    //
+    // {
+    //    1: {
+    //      1: { onActivate: callbackIdObject }
+    //    }
+    // }
+    this._eventCallbackData = {};
+
```

Comment:
Agree we need a bit more context to more easily understand what this means. I think I understand now based on the _setCallbackDataPerNode function but it would be more clear if you explicitly state at the start of this comment black that _eventCallbackData is structured as follows to allow callback IDs to be looked up using prismId, nodeId, and eventName.

Reply:
👍 I will add more details to the explanation.

Reply:
The following explanation has been added:
```
The member '_eventCallbackData' should store the callbackId for each node for each prism
PlatformFactory uses the callbackIds to unsubscribe all the callbacks when closing the prism
```